vtex/brand-ui
# Create Footer and Landing Footer

#### What is the purpose of this pull request?
As the title says

#### What problem is this solving?
[Task description](https://vtex-dev.atlassian.net/jira/software/projects/PHRS/boards/203?selectedIssue=PHRS-84)
[Footer examples on Figma](https://www.figma.com/file/w3uKnfM6ELfilCatuYOVMe/Brand-UI?node-id=0%3A1)

#### How should this be manually tested?
Run `yarn` and `yarn storybook`. Then, on `http://localhost:6006/`, it'll have two components: `Footer` and `FooterLanding`

Check the [live preview](https://deploy-preview-12--vtex-brand-ui.netlify.app/) of the components on Storybook.

#### Types of changes
